### PR TITLE
Minor API update

### PR DESCRIPTION
**Pull request by pablogs9**
RELATED:
- [ ] https://github.com/eProsima/Micro-XRCE-DDS-docs/pull/58
- [ ] https://github.com/eProsima/Micro-XRCE-DDS/pull/69
- [ ] https://github.com/micro-ROS/rmw-microxrcedds/pull/108

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Linux/236/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Windows/246/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Linux/237/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Windows/247/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Linux/238/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Windows/248/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Linux/239/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Windows/249/)

File: include/uxr/client/core/session/write_access.h [hunk 1]
```diff
@@ -74,10 +93,10 @@ uint16_t uxr_buffer_reply(
  * @param topic_size        The size of the topic in bytes.
  * @return A `bool` that identifies the request made by the Client, if the message can be serialized
```
Comment (jamoralp) [inline include/uxr/client/core/session/write_access.h:94]: change docs retval

File: include/uxr/client/core/session/write_access.h [hunk 2]
```diff
@@ -96,7 +115,7 @@ UXRDLLAPI bool uxr_prepare_output_stream(
  * @return A `bool` that identifies the request made by the Clientif the message can be serialized
```
Comment (jamoralp) [inline include/uxr/client/core/session/write_access.h:115]: change docs retval

Comment (pablogs9) [reply include/uxr/client/core/session/write_access.h]: Done

File: include/uxr/client/core/session/write_access.h (hunk 1 above)
Comment (pablogs9) [reply include/uxr/client/core/session/write_access.h]: Done

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Linux/243/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Windows/253/)

**Review by jamoralp — APPROVED** [review]

**Pull request merged**